### ISSUE TITLE
Internal Server Error when loaded

### ISSUE THREAD

**Issue by anthonylavado** [jellyfin/jellyfin-plugin-autoorganize#5]
So I installed the Auto Organize plug-in. Every time I go to it's page, I get an Internal Server Error on the Activity Log. The other pages appear to be fine.

```
 [2019-02-22 01:03:31.547 -05:00] [ERR] Error processing request
System.NullReferenceException: Object reference not set to an instance of an object.
   at Emby.AutoOrganize.Core.FileOrganizationService.GetResults(FileOrganizationResultQuery query)
   at Emby.AutoOrganize.Api.FileOrganizationService.Get(GetFileOrganizationActivity request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.Execute(Type serviceType, IRequest request, Object instance, Object requestDto, String requestName) in /jellyfin/Emby.Server.Implementations/Services/ServiceExec.cs:line 88
   at Emby.Server.Implementations.Services.ServiceController.Execute(HttpListenerHost appHost, Object requestDto, IRequest req) in /jellyfin/Emby.Server.Implementations/Services/ServiceController.cs:line 177
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost appHost, IRequest httpReq, IResponse httpRes, ILogger logger, String operationName, CancellationToken cancellationToken) in /jellyfin/Emby.Server.Implementations/Services/ServiceHandler.cs:line 95
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken) in /jellyfin/Emby.Server.Implementations/HttpServer/HttpListenerHost.cs:line 608
```

**Comment by anthonylavado** [comment]
To be clear for anyone that sees this - you can click “Got It” on the message that pops up, and continue.

I haven’t tested the functions themselves, but the rest of the UI seems to work.

**Comment by ssbarnea** [comment]
I confirm the same error and it should be treated as serious.

**Comment by MrNuggelz** [comment]
This is fixed by #10
